conf/empire/empire.yaml

```diff
@@ -129,46 +129,54 @@ stacks:
       InternalZoneName: vpc::InternalZoneName
       InternalHostname: empire-db
   - name: empireMinion
-    class_path: stacker_blueprints.empire.empire_minion.EmpireMinion
+    class_path: stacker_blueprints.empire.minion.EmpireMinion
     parameters:
       << : *vpc_parameters
       << : *docker_params
       InstanceType: ${empire_minion_instance_type}
-      MinSize: ${empire_minion_min_instance_count}
-      MaxSize: ${empire_minion_max_instance_count}
+      MinHosts: ${empire_minion_min_instance_count}
+      MaxHosts: ${empire_minion_max_instance_count}
       SshKeyName: ${ssh_key_name}
       ImageName: empire
       DisableStreamingLogs: ${empire_disable_streaming_logs}
   - name: empireController
-    class_path: stacker_blueprints.empire.empire_controller.EmpireController
+    class_path: stacker_blueprints.empire.controller.EmpireController
     parameters:
       << : *vpc_parameters
       << : *docker_params
-      ExternalDomain: ${external_domain}
-      InternalZoneId: vpc::InternalZoneId
       InstanceType: ${empire_controller_instance_type}
-      MinSize: ${empire_controller_min_instance_count}
-      MaxSize: ${empire_controller_max_instance_count}
+      MinHosts: ${empire_controller_min_instance_count}
+      MaxHosts: ${empire_controller_max_instance_count}
       SshKeyName: ${ssh_key_name}
       ImageName: empire
+      DatabaseSecurityGroup: empireDB::SecurityGroup
+  - name: empireDaemon
+    class_path: stacker_blueprints.empire.daemon.EmpireDaemon
+    parameters:
+      << : *vpc_parameters
+      << : *docker_params
+      DesiredCount: 1
+      TaskMemory: 256
+      TaskCPU: 256
```

Comment:
might as well go 1024 here, in theory this is the main purpose of the host, and that just means it can compete equally with other containers